caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.app.js

```diff
@@ -1575,6 +1627,24 @@
       }
     }
 
+    /**
```

Comment:
Please just delete this if it's not used.

Reply:
Deleted